components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadManager.kt

```diff
@@ -52,6 +55,14 @@ class DownloadManager(
         cookie: String = ""
     ): Long {
 
+        if (download.hasAValidProtocol()) {
```

Comment:
nit: `isSupportedProtocol()` is a bit easier to read (`hasAValid" looks weird 😃)

Reply:
👍